### DISCUSSION TITLE
Cannot create PDFs

### DISCUSSION THREAD

**Discussion by halloleo** [ickc/pancritic#17] [Q&A]
I have installed `pancritic `and `pypandoc `(and later `panflute `as well) into a virtual environment. `pandoc `itself is installed and on the path.

When I issue

    pancritic -t latex -o scr.pdf scr.md

I get the message

```
Cannot use pypandoc.
Use panflute instead.
Cannot use panflute.
Stop converting and output original format instead
```

and no output.

What's going wrong?

**Comment by ickc** [comment]
Sorry I completely missed this.

Can you try

```sh
which -a pandoc
which -a python
python -c 'import panflute'
python -c 'import pypandoc'
```

And paste the results here?

It seems like a PATH and/or PYTHONPATH issue.

**Reply by wilenius** [reply]
I have the same problem. All paths are working:

$ which -a python
/usr/bin/python
/bin/python
$ which -a pandoc
/usr/bin/pandoc
/bin/pandoc
$ python -c 'import panflute'
$ python -c 'import pypandoc'

Any ideas?

**Reply by ickc** [reply]
Can you provide a Minimal Working Example, such as the input text and command used?
